src/coreclr/pal/src/exception/seh-unwind.cpp

```diff
@@ -496,7 +496,9 @@ void GetContextPointers(unw_cursor_t *cursor, unw_context_t *unwContext, KNONVOL
 
```

Comment:
@janvorli, should this: https://github.com/dotnet/runtime/blob/f3e4e768d10dc502b0a7e9c417ae269dcf75dbc7/src/coreclr/pal/src/exception/seh-unwind.cpp#L186-L187 be changed to:
```cpp
    fpsimd_context* fp = GetNativeSigSimdContext(unwContext);
#define ASSIGN_FP_REG(reg) if (fp) *(NEON128*) &fp->vregs[reg] = winContext->V[reg];
```
(only pass one argument to `ASSIGN_FP_REG`)
While working on #64043, I had a misconfiguration (related to `UNWIND_CONTEXT_IS_UCONTEXT_T` detection) and found this issue when it compiled on linux arm64.

Reply:
@am11 Seems to be the case considering the only use of `ASSIGN_FP_REG` has a single argument.

https://github.com/dotnet/runtime/blob/f3e4e768d10dc502b0a7e9c417ae269dcf75dbc7/src/coreclr/pal/src/exception/seh-unwind.cpp#L134-L157

Reply:
I wonder if this code was ever compiling and is dead? cl.exe on Windows and clang on Linux both fail the `UNWIND_CONTEXT_IS_UCONTEXT_T` test and set it to 0 on `main`. To unveil the problem, i.e. misconfiguration which I had -- in order to set it to 1, delete the last `/include` from: https://github.com/dotnet/runtime/blob/f3e4e768d10dc502b0a7e9c417ae269dcf75dbc7/src/coreclr/pal/src/configure.cmake#L1033

Reply:
@am11 you are right, it is a dead code. I've added it when fixing floating point registers unwind because we had that code path for ARM64. However, the libunwind currently doesn't use the ucontext_t as unwind context for ARM64, IIRC a comment in there says it is done that way in order to reduce size of the unwind context. So everything for ARM64 with `UNWIND_CONTEXT_IS_UCONTEXT_T` set is not being ever compiled.
I think that we should just remove all of that and replace it with `#error`